sci-physics/pythia/pythia-8.3.13.ebuild

```diff
@@ -177,6 +203,11 @@ src_install() {
 		doins -r examples
 		docompress -x /usr/share/doc/${PF}/examples
 	fi
+	if use python; then
+		local site_dir=$(python_get_sitedir)
+		insinto "${site_dir#${EPREFIX}}"
+		doins lib/pythia8.so
```

Comment:
This would install the library into the sitedir itself. Usually things should be installed only into subdirectories of the sitedir.

Reply:
I have a bunch of top-level `.so`s, but I can move it (if it still works then):
```bash
 ls | grep "\.so"
3204bda914b7f2c6f497__mypyc.cpython-312-x86_64-linux-gnu.so
_blueman.so
_cffi_backend.cpython-312-x86_64-linux-gnu.so
datrie.cpython-312-x86_64-linux-gnu.so
_dbus_bindings.so
_dbus_glib_bindings.so
_fastjet.so
_fastjet.so.0
_fastjet.so.0.0.0
fontforge.so
libvirtmod.cpython-312-x86_64-linux-gnu.so
libvirtmod_lxc.cpython-312-x86_64-linux-gnu.so
libvirtmod_qemu.cpython-312-x86_64-linux-gnu.so
libxml2mod.so
psMat.so
pypamtest.so
pythia8.so
recollaspell.cpython-312-x86_64-linux-gnu.so
_ruamel_yaml.cpython-312-x86_64-linux-gnu.so
_snack.so
_ukkonen.abi3.so
```

Reply:
Yes we have more packages doing this, but it is considered a bug: https://projects.gentoo.org/python/guide/qawarn.html#stray-top-level-files-in-site-packages\

If the ebuild would use `distutils-r1.eclass` to manage the python part it would emit a QA warning about this.

The best solution (IMO) is to separate the python part from the main application and build that separately with `distutils-r1.eclass`. However that does not always work if the build system really intertwines the python code and the rest of the application.
```
src_$phase(){
      # main app $phase
      if use python; then
             cd python || die
              distutils-r1_src_$phase
     fi
}
```

Reply:
With `/usr/lib/python3.12/site-packages/pythia8/pythia8.so` I have to import it as 

```py
from pythia8 import pythia8
...
```

where as the normal/example/test way of importing is 

```py
import pythia8
...
```
corresponding to current behavior.